homeassistant/components/squeezebox/media_player.py

```diff
@@ -170,7 +169,7 @@ async def _discovered_player(player):
         ] = async_call_later(hass, DISCOVERY_INTERVAL, _discovery)
 
     _LOGGER.debug("Adding player discovery job for LMS server: %s", host)
-    asyncio.create_task(_discovery())
+    hass.async_create_task(_discovery())
```

Comment:
Doing this during startup means HA startup will block until this task is done running.

Reply:
I think that #88265 can help.

Reply:
Ah, good point. Converted PR to draft. I'll wait for #88265 to merge then refactor.

Reply:
It got merged!

Reply:
Was mass fixing them all and made a PR for squeezebox too: #88299